de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/auth/LoginServiceFactory.java

```diff
@@ -25,15 +32,39 @@ public LoginService getLoginService() {
 		
 		try {
 			service = (LoginService) Class.forName(serviceName).newInstance();
-		} catch (InstantiationException e) {
-			e.printStackTrace();
-		} catch (IllegalAccessException e) {
-			e.printStackTrace();
-		} catch (ClassNotFoundException e) {
-			e.printStackTrace();
+		} catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
+			Activator.getDefault().getLog().log(new Status(Status.ERROR, Activator.getPluginId(), Status.ERROR, "Couldn't create instance for " + ServerConfiguration.getLoginServiceClass(), e));
+		}
+		
+		if (service instanceof HashLoginService) {
+			
+			String filePath = null;
+			
+			// Check if the file exists, else try to resolve it in the bundle
+			if (new File((Activator.getDefault().getAuthFilePath())).exists()) {
+				filePath = Activator.getDefault().getAuthFilePath();

```

Comment:
Let's ask `ServerConfiguration` directly for this value, and if file doesn't exist, try to resolve it via Activator.